invenio_theme_tugraz/templates/invenio_theme_tugraz/header.html

```diff
@@ -32,6 +33,7 @@
                   </polyline>
                 </svg>
                 {{_ ("Home")}}
+              </span>
```

Comment:
Style: Normally those additions should be indented correctly.
Bugfix: Thumbs UP

Reply:
duly noted.